internal/pkg/cmd/prerunner.go

```diff
@@ -555,6 +567,10 @@ func (r *PreRun) AuthenticatedWithMDS(command *AuthenticatedCLICommand) func(*co
 			}
 		}
 
+		if err := r.setHubClient(command); err != nil {
```

Comment:
As per my comment above, this will get initialized even for commands that don't need it, and as we add clients over time, this function will grow linearly. We should copy the style of Kafka REST and write a `GetConfluentHubClient()` (or similarly named) function that instantiates it right before it's used.